gtk4/src/lib.rs

```diff
@@ -206,6 +206,18 @@ pub const TEXT_VIEW_PRIORITY_VALIDATE: u32 = ffi::GTK_TEXT_VIEW_PRIORITY_VALIDAT
 #[macro_use]
 mod rt;
 
+#[allow(dead_code)]
+#[cfg(test)]
+pub(crate) static TEST_THREAD_WORKER: once_cell::sync::Lazy<glib::ThreadPool> =
+    once_cell::sync::Lazy::new(|| {
```

Comment:
I believe this won't work on macOS (UI things have to happen from the thread that called `main()`), but otherwise this should be OK.

Reply:
interesting. well, i guess until we have a CI for macOS we can live with this :)